applications/ParticleMechanicsApplication/custom_conditions/particle_based_conditions/mpm_particle_base_condition.cpp

```diff
@@ -30,33 +30,33 @@ void MPMParticleBaseCondition::EquationIdVector(
 {
     KRATOS_TRY
 
-    GeometryType& rGeom = GetGeometry();
-    const unsigned int NumberOfNodes = rGeom.size();
-    const unsigned int dim = rGeom.WorkingSpaceDimension();
-    if (rResult.size() != dim * NumberOfNodes)
+    GeometryType& r_geometry = GetGeometry();
+    const unsigned int number_of_nodes = r_geometry.size();
+    const unsigned int dim = r_geometry.WorkingSpaceDimension();
+    if (rResult.size() != dim * number_of_nodes)
```

Comment:
sometimes it is Dimension, sometimes it is dimension and sometimes dim.

I know that there is no unique name. Should be something like working_space_dimension.